SingularityService/src/main/java/com/hubspot/singularity/config/MesosConfiguration.java

```diff
@@ -77,7 +77,7 @@
   private Optional<String> mesosUsername = Optional.empty();
   private Optional<String> mesosPassword = Optional.empty();
 
-  private long reconnectTimeoutMillis = 60000;
+  private long reconnectTimeoutMillis = 120000;
```

Comment:
This is lengthened because the attempt time now takes into account the startup period, not just the time from POST -> subscribed event like it did before